docs/modules/publisher.md

```diff
@@ -25,7 +25,7 @@ Container create options can be specified in the "Update IoT Edge Module" page o
 {
     "Hostname": "opcpublisher",
     "Cmd": [
-        "PkiRootPath=/mount/pki",
+        "--PkiRootPath=/mount/pki",
```

Comment:
Are you sure we need the '--' ? PkiRootPat does not have alternative command

Reply:
Yes, we do need it to avoid a warning regarding unknown option. Please note, that even with a warning the value will be applied in this case. The warning arises because even though we have added a parsing for `PkiRootPath` key, the parsing still looks for `-` or `--` in front of the options.

With this change we will also have one clear usage pattern for command line options which people should use.

Reply:
I've added `--` in front of `PkiRootPath` in other samples as well.